src/com/hrabrov/introduction_to_java/algorithmization/multidimensional_array/MultidimensionalArray.java

```diff
@@ -0,0 +1,206 @@
+        System.out.println("Original matrix:\n" + stringHelper.matrixToString(matrixForTask02));
+        System.out.println("First diagonal elements: " + stringHelper.arrayToString(resultOfTask02.firstDiagonal));
+        System.out.println("Second diagonal elements: " + stringHelper.arrayToString(resultOfTask02.secondDiagonal));
+
+        int[][] matrixForTask03 = new int[][]{
+                {1, 2, 3, 4, 5},
+                {6, 7, 8, 9, 10},
+                {11, 12, 13, 14, 15}
+        };
+        int[][] resultOfTask03 = Task03.displaySelectedLineAndColumn(matrixForTask03, 1, 4);
+        System.out.println("Result of task 03:");
+        System.out.println("Original matrix:\n" + stringHelper.matrixToString(matrixForTask03));
+        System.out.println("Selected row: " + stringHelper.arrayToString(resultOfTask03[0]));
+        System.out.println("Selected column: " + stringHelper.arrayToString(resultOfTask03[1]));
+
+        System.out.println("Result of task 04: ");
+        int[][] resultOfTask04 = Task04.createMatrixForTask04(8);
+        if (resultOfTask04.length != 0) {
+            System.out.println(stringHelper.matrixToString(resultOfTask04));
+        }
+
+        System.out.println("Result of task 05: ");
+        int[][] resultOfTask05 = Task05.createMatrixForTask05(4);
+        if (resultOfTask05.length != 0) {
+            System.out.println(stringHelper.matrixToString(resultOfTask05));
+        }
+
+        System.out.println("Result of task 06: ");
+        int[][] resultOfTask06 = Task06.createMatrixForTask06(4);
+        if (resultOfTask06.length != 0) {
+            System.out.println(stringHelper.matrixToString(resultOfTask06));
+        }
+
+        System.out.println("Result of task 07: ");
+        Task07.MatrixAndHerPositiveElements resultOfTask07;
+        resultOfTask07 = Task07.createMatrixForTask07(6);
+        System.out.println(stringHelper.matrixToString(resultOfTask07.matrix));
+        System.out.println("Quantity positive elements = " + resultOfTask07.quantityPositiveElements);
+
+        int[][] matrixForTask08 = new int[][]{
+                {0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
+                {0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
+                {0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
+                {0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
+        };
+        System.out.println("Result of task 08: ");
+        System.out.println("Original matrix: ");
+        System.out.println(stringHelper.matrixToString(matrixForTask08));
+        System.out.println("Starting replace column");
+        int firstNumber;
+        int secondNumber;
+        System.out.print("Enter number of first column for replace: ");
+        firstNumber = inputIntNumber();
+        System.out.print("Enter number of second column for replace: ");
+        secondNumber = inputIntNumber();
+        int[][] resultOfTask08 = Task08.replacedTwoColumn(matrixForTask08, firstNumber, secondNumber);
+        if (resultOfTask08.length != 0) {
```

Comment:
Always follow happy path for condition, you expect this length to be not 0, you need to check if it's zero and show error and proceed otherwise